client/lib/plans/index.js

```diff
@@ -14,6 +16,25 @@ import {
 	isFreeJetpackPlan,
 	isJetpackPlan
 } from 'lib/products-values';
+import { featuresList } from './constants';
+import sitesList from 'lib/sites-list';
+
+export function getValidFeatureKeys() {
+	return Object.keys( featuresList );
+}
+
+export function getSelectedSitePlanSlug() {
+	const site = sitesList().getSelectedSite();
+	return get( site, 'plan.product_slug' );
+}
+
+export function hasFeature( feature ) {
```

Comment:
This looks nice, but I think we should use `getSite( siteId )` and the in the nudge component we would do `hasFeature( feature, getSelectedSite.ID )`.